### PR TITLE
calculator 

### PR DESCRIPTION
**Pull request by achal-webonise**

### CODE REVIEW

File: Calculator/calculator.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    <title>Calculator</title>
+    
+  </head>
+ 
+    
+        <body> 
+          <script src="calculator.js">
+          </script>
+</body>
+</html>
```
Comment (aashiq-webonise) [inline Calculator/calculator.html]: Add End of line

File: Calculator/calculator.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    setNum1("");
+    setNum2("");
+    setOperator("");
+    setResult("");
+  };
+
+  const getButtons = () => {
+    if (mode === "simple") {
+      return [
+        { label: "1", onClick: () => handleNumberClick("1") },
```
Comment (aashiq-webonise) [inline Calculator/calculator.js]: Please Create a generic Function , maybe use an array for your numbers , this approach seems redundant

File: Calculator/calculator.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
+import React, { useState } from "eact";
```
Comment (aashiq-webonise) [inline Calculator/calculator.js]: This is one way of doing it , but please use Create React app for the same

File: node_modules/.bin/acorn
Comment (swapnil-webonise) [inline node_modules/.bin/acorn:1]: We should not commit node_modules folder, it should be in git ignore

**Review by swapnil-webonise — CHANGES_REQUESTED** [review]

**Pull request closed without merging**